src/networks/abc/chainparamsconstants.cpp

```diff
@@ -5,8 +5,8 @@
 #include <chainparamsconstants.h>
 
 namespace ChainParamsConstants {
-    const BlockHash MAINNET_DEFAULT_ASSUME_VALID = BlockHash::fromHex("00000000000000005ce0908e5a27e2f4c202f9f44445afff0899d254439f8480");
-    const uint256 MAINNET_MINIMUM_CHAIN_WORK = uint256S("00000000000000000000000000000000000000000153f1c5a543971b7f71bff5");
+    const BlockHash MAINNET_DEFAULT_ASSUME_VALID = BlockHash::fromHex("00000000000000001f9f06b89da888cfbb697a3c06251031e46e1c5f9cab259d");
```

Comment:
Probably wrong, not sure if we had updated this before? @EyeOfPython

Reply:
No, we haven't.